apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -290,6 +292,25 @@ export const Timezones = {
 
 async function addHostsToDb(eventTypes: InputEventType[]) {
   for (const eventType of eventTypes) {
+    // Create host groups first if they exist
+    const groupIdMapping: Record<string, string> = {};
+    if (eventType.hostGroups?.length) {
+      for (const group of eventType.hostGroups) {
+        const createdGroup = await prismock.hostGroup.create({
```

Comment:
Should we do this as a create many? NIT - its only a scenario not too bothered if this doesnt get changed